contrib/docker/docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt update && apt upgrade -yq && \
   gfortran git libblas-dev liblapack-dev libopenmpi-dev \
   lsb-release ninja-build numdiff openmpi-bin \
   openmpi-common wget zlib1g-dev \
-  libboost-all-dev &&\
+  libboost-all-dev lcov &&\
```

Comment:
this is not the correct file, it needs to go into `contrib/docker/tester/Dockerfile`.